src/client.cpp

```diff
@@ -596,7 +603,7 @@ bool CanPauseStream(void)
 
 bool IsRealTimeStream()
 {
-  return true;
+  return m_bLivePlayback;
```

Comment:
Are this old fashion APIs even used when there is the stream property (`isrealtimestream`) set?

Reply:
Currently both are used. The property for kodi’s internal inputstreams and this function for inputstream addons.

This will be cleaned up in the future when we move PVRs inputstreams into their own addons.

Reply:
I must rebase this.

Reply:
> I must rebase this.

No. I don't want other global variable... I'll reimplemented it already...

And how about those other Is\*, Can\*() methods?

Reply:
...and another thought... can we just simply return `false` here to get the seek buttons all the time?

Reply:
Having the seek buttons is useless unless the underlying inputstream supports them.

Reply:
You can leave the Is* Can* functions. As I said this will get cleaned up once the inputstream work is done.

Reply:
Don't forget you need to reimplement this for the Leia branch too ;)

Reply:
> Don't forget you need to reimplement this for the Leia branch too ;)

That's just simple cherry-pick ;)